docs/test-pages/TCD-002.adoc

```diff
@@ -0,0 +1,33 @@
+= TCD-002
+
+== Purpose
+
+Checking that the IDE knows the jsl file extension as a JSL file.
+
+=== Tested requirements
+
+REQ-SYNT-001, REQ-MDL-001
+
+=== Used JSL model
+
+xref:resources/m1.jsl[m1.jsl], xref:resources/m1.txt[m1.txt]
+
+== Test scenario
+
+. Start the IDE (Eclipse or idea).
+
+. Open the xref:resources/m1.txt[m1.txt].
+
+. Verify that the IDE doesn't colour the text, because it is handle the file as a simple text file.
+
+. Open the xref:resources/m1.jsl[m1.jsl].
+
+. Verify that the IDE colours the text, because it handles the file as a standard JSL file.
+
+. Goto the end of the file, and write in a new line the "ent" string, and press the CTRL + SPACE keys on the keyboard.
+
+. The editor complements the "ent" to "entity" and colours it as a keyword.
+
+. Close both file, and do not save them.
```

Comment:
do not save them ?

Reply:
Yes. Because, this test case verifies that the IDE works properly. If save the modification, the next test cycle you cannot do the steps properly. And the saving mechanism is a existing function in the IDE that function is out of scope in this context.

Reply:
We should not test IDE specific functionality. IDE plugins are okay, but not stuff like this for sure!

Reply:
I don't know what is the different between the IDE specific fuctionality and the IDE plugins. I think if we create a plugin that supports the jsl editing, we should test it. Or not?